knowledge/geography/highways/interstate_84/qna.yaml

```diff
@@ -0,0 +1,54 @@
+created_by: makelinux
+domain: geography
+seed_examples:
+  - question:
+      What are some notable geographical features and landmarks along
+      Interstate 84's route?
+    answer:
```

Comment:
Thank you for your submission; taking a look at your `qna.yaml`, we noticed that you have some issues with the multiline `yaml` syntax.

Can you take a look at: https://yaml-multiline.info/ and fix up the file? Thanks!

Reply:
Sorry, which issue? Plain flow scalar allows quotes.